cmd/waku/server/rpc/relay.go

```diff
@@ -150,7 +151,11 @@ func (r *RelayService) GetV1AutoMessages(req *http.Request, args *TopicArgs, rep
 		return err
 	}
 	select {
-	case msg := <-sub.Ch:
+	case msg, open := <-sub.Ch:
+		if !open {
+			r.log.Error("consume channel is closed for subscription", zap.String("pubsubTopic", args.Topic))
+			return errors.New("consume channel is closed for subscription")
```

Comment:
Probably makes sense to extract this error to a global var, and use it in here and in line 218